cmd/dev/ci/deps/main.go

```diff
@@ -0,0 +1,19 @@
+package deps
+
+import (
+	"github.com/spf13/cobra"
+)
+var pOs string
+var pArch string
+var pConfig string
+
+var Main = &cobra.Command{
+	Use:   "deps",
+	Short: "Helpers for binary dependencies in Makefiles.",
+}
+
+func init() {
+	Main.PersistentFlags().StringVarP(&pOs, "os", "o", "","OS the binary should run on. Currently only 'linux' and 'darwin' are supported.")
```

Comment:
To avoid globals you can use `StringP()` and access the var later with `github.com/ory/x/flagx.MustGetString()`

Reply:
Oh I like it... I blindly copied what I have seen in other examples... But I didnt like the use of globals. Cool.